test/test_script.py

```diff
@@ -1568,3 +1569,45 @@ def test_cmd_unique(self):
             self.parser.eval("$unique()")
         with self.assertRaisesRegex(ScriptError, areg):
             self.parser.eval("$unique(B:AB; D:C; E:D; A:A; C:X,1,:,extra)")
+
+    def test_cmd_countryname(self):
+        from picard import i18n
+        context = Metadata()
+        context["foo"] = "ca"
+        context["bar"] = ""
+        context["baz"] = "INVALID"
+
+        # Test with Russian locale
+        i18n.setup_gettext('build/locale', ui_language='ru')
```

Comment:
I think the calls to i18n.setup_gettext can be removed, since this is now mocked

Reply:
I believe that the i18n calls are still required (at least once) to add `gettext_countries` to `builtins`.  I get an `AttributeError` exception if I try running the tests locally with them removed.

Reply:
Ah, yes. I think that's right. Other tests also need to initialize this. They usually use `setup_gettext(self.localedir, "C")`, but since we are using locale specific stuff here let's keep it the way it is

Reply:
I was able to resolve this by removing the `i18n` calls and manually add `gettext_countries` to `builtins` if it wasn't found.

Reply:
Rebased to remove the typo commit.